tests/simple/main.cpp

```diff
@@ -481,7 +481,7 @@ static void RunTests(Client& client) {
     ArrayExample(client);
     CancelableExample(client);
     DateExample(client);
-    DateTime64Example(client);
+//    DateTime64Example(client);
```

Comment:
I also forgot to commit uncommented line.

I had commented it because this example always fails [there](https://github.com/ManagedDatabases/clickhouse-cpp/blob/master/tests/simple/main.cpp#L151).

Reply:
Ok, you are right, but I am not sure it's your problem, so you can keep it unchanged for now